programs/util.h

```diff
@@ -84,7 +84,9 @@ extern "C" {
 /*-**************************************************************
 *  Basic Types
 *****************************************************************/
-#if  !defined (__VMS) && (defined (__cplusplus) || (defined (__STDC_VERSION__) && (__STDC_VERSION__ >= 199901L) /* C99 */) )
+#ifndef BASIC_TYPES_DEFINED
```

Comment:
This is just for LZ4. I've done similar changes for LZ4 and I want to use the same `util.h` and `platform.h`. If you'll decided to join them it's also not a problem.